README.md

```diff
@@ -130,11 +130,22 @@ Example: `partners_count: 4`
 Fractions of the original dataset each partner receives to mock a collaborative ML scenario where each partner provides data for the ML training.  
 Example: `amounts_per_partner: [0.3, 0.3, 0.1, 0.3]`
 
-`samples_split_option`: `random` or `stratified`  
+`samples_split_option`: `random` (default) or `stratified`  
 How the original dataset data samples are split among partners:
 
-- `random`: the dataset is shuffled and the split is done randomly, ensuring a homogeneous data distribution among partners
-- `stratified`: the dataset is stratified per labels, so that partners have datasets covering different regions of space
+- `random`: the dataset is shuffled and partners receive data samples selected randomly
+- `stratified`: the dataset is stratified per labels and each partner receives certain classes only
```

Comment:
I would keep only `labels `or `class`, since in this case, these two words have the same meaning
- `stratified`: the dataset is stratified per class and each partner receives certain classes only

Reply:
Done